src/test/groovy/pl/nextleveldev/smart_route/ApplicationContextLoadSpec.groovy

```diff
@@ -0,0 +1,23 @@
+package pl.nextleveldev.smart_route
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.context.ApplicationContext
+import spock.lang.Specification
+
+@SpringBootTest
+class ApplicationContextLoadSpec extends Specification {
+
+    @Autowired
+    ApplicationContext context
+
+    @Value('${spring.application.name}')
```

Comment:
1. Interpolation of Strings in Groovy.
2. Escaping `${}` in `@Value`.
3. Single quotes indicate: `${}` as plain text.

In Groovy and Spock, it is better to use double quotation marks:
```suggestion
    @Value("\${spring.application.name}")
```

In addition, make sure that `application.yml` (or `application.properties`) the `spring.application.name` is defined.